eos-parental-controls-client/eos-parental-controls-client.py

```diff
@@ -164,6 +192,18 @@ def main():
     parser_check.add_argument('path',
                               help='path to a program to check')
 
+    # ‘oars-section’ command
+    parser_oars_section = subparsers.add_parser('oars-section',
+                                                parents=[common_parser],
+                                                help='get the value of a '
+                                                     'given OARS section')
+    parser_oars_section.set_defaults(function=command_oars_section)
+    parser_oars_section.add_argument('user', default='', nargs='?',
```

Comment:
I think it's weird to have an optional positional parameter that's not at the end of the parameter list.

Reply:
Yeah, but I thought it was a bit weirder to provide the OARS section before the username. At least this way round it’s kind of hierarchical — query the user first, then query the OARS component of their accountsservice data.